faunadb-scala/src/main/scala/faunadb/streaming/DocumentValueFlowProcessor.scala

```diff
@@ -0,0 +1,87 @@
+package faunadb.streaming
+
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.{Flow, SubmissionPublisher}
+
+import faunadb.values._
+
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+private [faunadb] class DocumentValueFlowProcessor(loadDocument: () => Future[Value])(implicit ec: ExecutionContext)
```

Comment:
Is this scala class required for the java implementation?

Reply:
I have not added the snapshot query helper yet. 
This means I will create a Java version of this class :)

Reply:
here it is https://github.com/fauna/faunadb-jvm/pull/275/commits/800691d9475d210de64b2402a456200a91ea89d0

Reply:
In theory, we should not be seeing this class in this PR here, right? Maybe this branch needs to be updated with the latest changes from the Scala Streaming branch (`streaming`)?

Reply:
Oh I finally understand Erick comment :face_with_head_bandage: 
You are right, something went bad during the merge most likely due to the class renaming.
I will clean that up, thanks!

Reply:
https://github.com/fauna/faunadb-jvm/pull/275/commits/b08ce6dc849f9a9caedfa3b094f38ce1c975060a :broom: